jetbrains: 2025.1 -> 2025.1.2

JBassett opened the pull request:
jetbrains: 2025.1 -> 2025.1.2

jetbrains.clion: 2025.1 -> 2025.1.1
jetbrains.datagrip: 2025.1 -> 2025.1.2
jetbrains.gateway: 2025.1 -> 2025.1.1
jetbrains.goland: 2025.1 -> 2025.1.1
jetbrains.idea-community: 2025.1 -> 2025.1.1.1
jetbrains.idea-ultimate: 2025.1 -> 2025.1.1.1
jetbrains.mps: 2024.3.2 -> 2025.1
jetbrains.phpstorm: 2025.1 -> 2025.1.1
jetbrains.pycharm-community: 2025.1 -> 2025.1.1
jetbrains.pycharm-professional: 2025.1 -> 2025.1.1
jetbrains.rider: 2025.1 -> 2025.1.2
jetbrains.ruby-mine: 2025.1 -> 2025.1.1
jetbrains.rust-rover: 2025.1 -> 2025.1.2
jetbrains.webstorm: 2025.1 -> 2025.1.1

jetbrains.plugins: update

Validated Ultimate with a multi module project with kotlin, python, and react.
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

JBassett commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 405101`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 7 packages built:</summary>
  <ul>
    <li>jetbrains.clion</li>
    <li>jetbrains.datagrip</li>
    <li>jetbrains.idea-community-bin</li>
    <li>jetbrains.idea-ultimate</li>
    <li>jetbrains.phpstorm</li>
    <li>jetbrains.rider</li>
    <li>jetbrains.rust-rover</li>
  </ul>
</details>

ners commented:
If you run the update script again you'll also get:
```
jetbrains.webstorm: 2025.1 -> 2025.1.1
```

JBassett commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 405101`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 11 packages built:</summary>
  <ul>
    <li>jetbrains.clion</li>
    <li>jetbrains.datagrip</li>
    <li>jetbrains.idea-community-bin</li>
    <li>jetbrains.idea-ultimate</li>
    <li>jetbrains.phpstorm</li>
    <li>jetbrains.pycharm-community-bin</li>
    <li>jetbrains.pycharm-professional</li>
    <li>jetbrains.rider</li>
    <li>jetbrains.ruby-mine</li>
    <li>jetbrains.rust-rover</li>
    <li>jetbrains.webstorm</li>
  </ul>
</details>

aoli-al commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 405101`

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 11 packages built:</summary>
  <ul>
    <li>jetbrains.clion</li>
    <li>jetbrains.datagrip</li>
    <li>jetbrains.idea-community (jetbrains.idea-community-bin)</li>
    <li>jetbrains.idea-ultimate</li>
    <li>jetbrains.phpstorm</li>
    <li>jetbrains.pycharm-community (jetbrains.pycharm-community-bin)</li>
    <li>jetbrains.pycharm-professional</li>
    <li>jetbrains.rider</li>
    <li>jetbrains.ruby-mine</li>
    <li>jetbrains.rust-rover</li>
    <li>jetbrains.webstorm</li>
  </ul>
</details>

aoli-al reviewed (approved):
verified idea-ultimate build on macOS

Linschlager commented:
> Thanks! I'm really hoping that [this](https://github.com/NixOS/nixpkgs/pull/399364#issuecomment-2847160108) bug is fixed by the update.

@phip1611 I still get this issue on 25.1.1 and have to manually Ctrl-C and zsh to get the right terminal.

theCapypara commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 405101`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 11 packages built:</summary>
  <ul>
    <li>jetbrains.clion</li>
    <li>jetbrains.datagrip</li>
    <li>jetbrains.idea-community-bin</li>
    <li>jetbrains.idea-ultimate</li>
    <li>jetbrains.phpstorm</li>
    <li>jetbrains.pycharm-community-bin</li>
    <li>jetbrains.pycharm-professional</li>
    <li>jetbrains.rider</li>
    <li>jetbrains.ruby-mine</li>
    <li>jetbrains.rust-rover</li>
    <li>jetbrains.webstorm</li>
  </ul>
</details>

ners reviewed (changes requested):
Please run the update again to get the latest versions:
```
jetbrains.datagrip: 2025.1 -> 2025.1.2
jetbrains.rider: 2025.1 -> 2025.1.2
```
Also please remove the old commits so that this PR only has the two generated by the script.

jamesward commented:
Tested idea-ultimate on NixOS and works great. Thanks!

JBassett commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 405101`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 11 packages built:</summary>
  <ul>
    <li>jetbrains.clion</li>
    <li>jetbrains.datagrip</li>
    <li>jetbrains.idea-community-bin</li>
    <li>jetbrains.idea-ultimate</li>
    <li>jetbrains.phpstorm</li>
    <li>jetbrains.pycharm-community-bin</li>
    <li>jetbrains.pycharm-professional</li>
    <li>jetbrains.rider</li>
    <li>jetbrains.ruby-mine</li>
    <li>jetbrains.rust-rover</li>
    <li>jetbrains.webstorm</li>
  </ul>
</details>

theCapypara reviewed (approved):
LGTM, thanks! - Also checked with the new [tests](https://github.com/NixOS/nixpkgs/pull/403994).

jamesward commented:
More new versions today (including ultimate 2025.1.1.1) so please re-run the update script.

JBassett commented:
While I don't mind re-running this, shouldn't we try to get this merged and create new PRs for future updates?  Is there something I'm missing in order to get this merged?

jamesward commented:
Yeah, it'd be great to get this merged. But while we wait, let's keep it updated.

theCapypara commented:
From my point of view this is fine to be merged, but I don't have permissions. I think leona and most other contributors with merge permissions are currently pretty occupied with getting NixOS 25.04 out, so please be patient.

JBassett commented:
Not trying to rush things or sound impatient, just making sure it's not something I'm doing that's causing the delay.

jamesward commented:
Note that some plugins that use JCEF (markdown, Amazon Q, etc) we may need to bump JCEF as they seem currently broken. PR for that: https://github.com/NixOS/nixpkgs/pull/406654
If timing works, we could potentially update the JDK / JCEF along with this PR.

ners commented:
@JBassett I'm sorry to trouble you again. If you run the update again you'll get another update in:
```
jetbrains.phpstorm: 2025.1 -> 2025.1.1
```

JBassett committed:
jetbrains: 2024.3.2 -> 2025.1.2

jetbrains.clion: 2025.1 -> 2025.1.1
jetbrains.datagrip: 2025.1 -> 2025.1.2
jetbrains.gateway: 2025.1 -> 2025.1.1
jetbrains.goland: 2025.1 -> 2025.1.1
jetbrains.idea-community: 2025.1 -> 2025.1.1.1
jetbrains.idea-ultimate: 2025.1 -> 2025.1.1.1
jetbrains.mps: 2024.3.2 -> 2025.1
jetbrains.phpstorm: 2025.1 -> 2025.1.1
jetbrains.pycharm-community: 2025.1 -> 2025.1.1
jetbrains.pycharm-professional: 2025.1 -> 2025.1.1
jetbrains.rider: 2025.1 -> 2025.1.2
jetbrains.ruby-mine: 2025.1 -> 2025.1.1
jetbrains.rust-rover: 2025.1 -> 2025.1.2
jetbrains.webstorm: 2025.1 -> 2025.1.1

JBassett committed:
jetbrains.plugins: update

JBassett commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 405101`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 14 packages built:</summary>
  <ul>
    <li>jetbrains.clion</li>
    <li>jetbrains.datagrip</li>
    <li>jetbrains.gateway</li>
    <li>jetbrains.goland</li>
    <li>jetbrains.idea-community-bin</li>
    <li>jetbrains.idea-ultimate</li>
    <li>jetbrains.mps</li>
    <li>jetbrains.phpstorm</li>
    <li>jetbrains.pycharm-community-bin</li>
    <li>jetbrains.pycharm-professional</li>
    <li>jetbrains.rider</li>
    <li>jetbrains.ruby-mine</li>
    <li>jetbrains.rust-rover</li>
    <li>jetbrains.webstorm</li>
  </ul>
</details>
